SDP: fix compilation warnings

Fix the following compilation warning:
/tmp/dotan/ofed_kernel/drivers/infiniband/ulp/sdp/sdp_tx.c: In function 'sdp_post_send':
/tmp/dotan/ofed_kernel/drivers/infiniband/ulp/sdp/sdp_tx.c:124: warning: cast from pointer to integer of different size

Signed-off-by: Dotan Barak <dotanb@dev.mellanox.co.il>

---

 drivers/infiniband/ulp/sdp/sdp_tx.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/infiniband/ulp/sdp/sdp_tx.c b/drivers/infiniband/ulp/sdp/sdp_tx.c
index 3e17b96..4e369da 100644
--- a/drivers/infiniband/ulp/sdp/sdp_tx.c
+++ b/drivers/infiniband/ulp/sdp/sdp_tx.c
@@ -121,7 +121,7 @@ void sdp_post_send(struct sdp_sock *ssk, struct sk_buff *skb)
 
 	if (skb->len <= ssk->inline_thresh && !skb_shinfo(skb)->nr_frags) {
 		SDPSTATS_COUNTER_INC(inline_sends);
-		sge->addr = (u64) skb->data;
+		sge->addr = (u64)(unsigned long)skb->data;
 		sge->length = skb->len;
 		sge->lkey = 0;
 		frags = 0;
